dashboard/config/upstream-usb.config

```diff
@@ -10,7 +10,7 @@ CONFIG_DEBUG_AID_FOR_SYZBOT=y
 
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 5.5.0-rc2 Kernel Configuration
+# Linux/x86 5.5.0-rc5 Kernel Configuration
```

Comment:
@xairy PTAL
I've updated the usb config with the new scripts. There are some diffs. Please double check.